gammapy/spectrum/spectrum_extraction.py

```diff
@@ -18,6 +16,7 @@
 from ..image import ExclusionMask
 from ..region import SkyCircleRegion, find_reflected_regions
 from ..utils.energy import EnergyBounds, Energy
+from ..irf import EffectiveAreaTable, EnergyDispersion
```

Comment:
The problem is the import here:
https://github.com/gammapy/gammapy/blob/master/gammapy/irf/exposure.py#L4

As a temp solution, you could move this into the exposure_cube function and things should work.

I'll think about a long-term proper solution and make a PR.